task/output.go

```diff
@@ -13,6 +13,13 @@ type Output struct {
 	output *bytes.Buffer
 }
 
+// PublishOutput specific output for publishing api
+type PublishOutput struct {
+	*Output
+	PublishDetail
+	bar string
```

Comment:
Hmm, maybe `bar` is not the most expressive name.. this is a progress bar, so could we rename it as that? or `barName` or similar?

Reply:
As comment below renamed to barType.